cicd/deploy_ephemeral_env.sh

```diff
@@ -11,5 +11,6 @@ bonfire deploy \
     --set-template-ref ${APP_NAME}/${COMPONENT_NAME}=${GIT_COMMIT} \
     --set-image-tag ${IMAGE}=${IMAGE_TAG} \
     --namespace ${NAMESPACE} \
+    --timeout ${DEPLOY_TIMEOUT:-300} \
```

Comment:
@bsquizz this isn't already exposed via an env variable, is it?

Reply:
There is not yet. But can we put this in the common deploy logic script and use the same syntax used here for setting the default value... Jeremy Audet made some cases for why using this syntax is better.

https://github.com/RedHatInsights/bonfire/blob/9668507e6b15469380f20a0322b73dcebf4a300a/cicd/_common_deploy_logic.sh#L15

Also can we add a comment up top explaining what the var does: for ex https://github.com/RedHatInsights/bonfire/blob/9668507e6b15469380f20a0322b73dcebf4a300a/cicd/_common_deploy_logic.sh#L5

Reply:
@bsquizz  done, updated! 
I saw more scripts that could potentially use this variable but we only need it here, and I don't want to create any unwanted side-effect